site/styles/_social-media-buttons.scss

```diff
@@ -7,7 +7,7 @@
 .social-media-buttons-title {
 	align-self: center;
 	padding-right: 10px;
-	h5 {
+	h2 {
```

Comment:
Should you leave h5 here for backwards compatibility?

Reply:
Yup. Have added.